docs/guides/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+```{include} ../../CONTRIBUTING.md
+:relative-docs: ../../
+```
```

Comment:
I am not fully sure it is easy to follow what this file and other files in the `docs/guides/` are for and for whom.
I appreciate they are populated when compiling documentation using Sphinx. But the whole repo looks extremely convoluted and is not easy to follow what is what where.

Should maybe tell the user when and with what this folder will be populated with? I really don't think it is easy to follow.

For instance, there is already a `CONTRIBUTING.md` file in the root directory for people who wants to contribute to the ukgovdatascience cookiecutter project. And then there is a `{{ cookiecutter.repo_name }}/CONTRIBUTING.md` which is up to the project owner to tweak as they wish and that is aimed at the contributor of that project.

Reply:
Honestly, I agree with you @exfalsoquodlibet - I believe it makes sense within the template itself, as there are Markdown files elsewhere that may be useful in the main Sphinx documentation (up for debate though!).

However, at the root-level it's getting very convoluted to get MySt working, based on the way it resolves relative links in Markdown. It also imposes some unreasonable restrictions on what type of links can/can't be written in included Markdown files, which I don't think are maintainable going forward.

I'm inclined to remove Sphinx out of the root documentation, and either leave bare Markdown files and/or build a Middleman-based website since we're the ones hosting the documentation! See the [GOV.UK Developer docs](https://docs.publishing.service.gov.uk/) for an example. This would be work for a future PR.

I've only included in this PR for now as it works (though convoluted), and I wanted to avoid duplication of Markdown files that may make maintainability more difficult, e.g. pre-commit hooks documentation applies to both `govcookiecutter` and its templates.

Would you be happy to leave this in for now, and deal with it later as tech debt? Or would you like me to fix it now?

Reply:
Hi Eric, yes, makes sense to leave this for now.